deploy/requirements.txt

```diff
@@ -0,0 +1 @@
+flask
```

Comment:
I have this file on my machine, but looks like I forgot to push it to Github.

Instead of 'flask', include the following two libraries as we also need pyyaml in kubeconfiggenerator. 

Flask
pyyaml